packages/@webex/contact-center/package.json

```diff
@@ -0,0 +1,66 @@
+{
+  "name": "@webex/contact-center",
+  "description": "",
+  "main": "dist/index.js",
+  "devMain": "src/index.ts",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/webex/webex-js-sdk.git",
+    "directory": "packages/@webex/contact-center"
+  },
+  "engines": {
+    "node": ">=20.x"
+  },
+  "scripts": {
+    "build:src": "webex-legacy-tools build -dest \"./dist\" -src \"./src\" -js -ts && yarn build",
+    "build": "yarn run -T tsc --declaration true --declarationDir ./dist/types",
+    "deploy:npm": "yarn npm publish",
+    "docs": "typedoc --emit none",
+    "fix:lint": "eslint 'src/**/*.ts' --fix",
+    "fix:prettier": "prettier \"src/**/*.ts\" --write",
+    "prebuild": "rimraf dist",
+    "test": "yarn test:style && yarn test:unit",
+    "test:style": "eslint 'src/**/*.ts'",
+    "test:unit": "webex-legacy-tools test --unit --runner jest"
+  },
+  "dependencies": {
+    "@types/platform": "1.3.4",
+    "@webex/contact-center": "workspace:*",
```

Comment:
Why is this self-dependency present?

Reply:
I have to import ContactCenter into the test file so I added it as a dependency. Observed the same in plugin-meetings as well. Will try running the test without the dependency and check

Reply:
Removed it